pkg/test/step.go

```diff
@@ -183,6 +186,32 @@ func (s *Step) Create(namespace string) []error {
 	return errors
 }
 
+// DoKubectl runs all kubectl commands defined for the test step.
+func (s *Step) DoKubectl(namespace string) []error {
+	errs := []error{}
+
+	if s.Step == nil || s.Step.Kubectl == nil {
+		return errs
+	}
+
+	for _, cmd := range s.Step.Kubectl {
+		stdout := &bytes.Buffer{}
+		stderr := &bytes.Buffer{}
+
+		s.Logger.Log("Running kubectl:", cmd)
+
+		err := testutils.Kubectl(context.TODO(), namespace, cmd, s.Dir, stdout, stderr)
```

Comment:
what is the difference between this DoKubectl and the one defined in harness?

Reply:
It's specific handling for the logging and errors related to steps versus the suite. I can probably abstract it out better, though.